.vuestorefrontcloud/docs/default.conf

```diff
@@ -7,6 +7,10 @@ server {
   }
 
   location /v2/ {
+    if ($request_uri ~ ^/(.*)\.html(\?|$)) {
+        return 302 /$1;
+    }
+    try_files $uri $uri.html $uri/ =404;
```

Comment:
I'm wondering about `$uri/` part - what does it do here?

Reply:
$uri/ matches the directory but I'm still looking for the latest issue so I'm converting to draft for now